apps/server/src/shared/domain/entity/lesson.entity.ts

```diff
@@ -23,7 +24,7 @@ export enum ComponentType {
 	ETHERPAD = 'Etherpad',
 	GEOGEBRA = 'geoGebra',
 	INTERNAL = 'internal',
-	LERNSTORE = 'resources',
+	RESOURCES = 'resources',
```

Comment:
Since the naming here is not explanatory, you should perhaps write a comment in the line.

```suggestion
	RESOURCES = 'resources', // lernstore
```